tests/unit/src/presenters/question-to-inset.test.js

```diff
@@ -1,45 +1,91 @@
 const presenters = require("../../../../src/presenters");
-const moment = require("moment");
+
+Date.now = jest.fn(() => new Date("2024-05-01"));
 
 describe("question-to-inset", () => {
   let translate;
   let question;
-  let locals;
-  const language = "en";
+  let data;
+  const englishLanguage = "en";
+  const welshLanguage = "cy";
+  const welshDate = "01 Chwefror 2024";
+  const englishDate = "01 February 2024";
```

Comment:
UCD will want it in the form `1 February 2024` (no leading 0s) so I think the moment format will need updating to `D MMMM YYYY` ?